### DISCUSSION TITLE
Hydra issues in Ray Cluster

### DISCUSSION THREAD

**Discussion by Jixi123** [facebookresearch/hydra#2951] [Q&A] [answered]
Hi! 
I was wondering if the hydra ray launcher was necessary to use Hydra at all in a Ray Cluster. Right now, when I try and import hydra inside an ec2 instance created by the ray cluster, I get an error that says something like: "Error importing generated parsers, run 'python setup.py antlr' to regenerate". This doesn't happen locally outside the cluster. Thanks! 

Sorry if this isn't the best place or have the best formatting, this is my first time using discussions.

**Comment by halfrot** [comment, marked as answer]
It may not be a ray cluster issue. I also ran into this problem and solved it by upgrading my Java. You can check what the error message says and run python setup.py antlr.
